polyfill/lib/ecmascript.mjs

```diff
@@ -39,7 +39,7 @@ import {
   NANOSECONDS
 } from './slots.mjs';
 
-const DAYMILLIS = 3600000;
+const DAYMILLIS = 86400000;
```

Comment:
How'd we get this wrong?

Reply:
:man_shrugging: it has test coverage now, so it should be okay from now on